control/plugin/cpolicy/node.go

```diff
@@ -71,6 +71,56 @@ func NewPolicyNode() *ConfigPolicyNode {
 	}
 }
 
+func (c *ConfigPolicyNode) CopyRules() ([]Rule, error) {
+	rules := make([]Rule, len(c.rules))
+	i := 0
+	for _, rule := range c.rules {
```

Comment:
`c.rules` is a `map[string]Rule`, so it's not possible to use `i` as key. I would need to create slice of map values firsts.